Fixed typo in help message of command 'nikola auto'.

campbellized opened the pull request:
I changed the description of argument -p from 'Port nummber' to 'Port number'

Kwpolska commented:
Thanks for your contribution!

1. It looks like your commit was misattributed. Please add the e-mail address you used for this commit [to your GitHub profile](https://github.com/settings/emails) so your commits will appear as yours.
2. This is a really minor change, with no effect on behavior of Nikola itself, and as such it should not be in CHANGES.txt.

campbellized commented:
Thank you for notifying me about my email address. It was a secondary email account but I just added it to my GitHub profile. I also removed my line item from CHANGES.txt.

Kwpolska commented:
Merged, thanks for contributing!
